persimmon/website/urls.py

```diff
@@ -23,8 +25,20 @@
     path('api/bankaccount/new', apis.create_bank_account, name='create-bank-account'),
     path('user-lookup', apis.user_lookup, name='user-lookup'),
 
-    path('reset-password', html_views.reset_password_page),
-    path('reset-password/sent', html_views.reset_password_sent),
+    path('', include('django.contrib.auth.urls')),
```

Comment:
Do we really need to include all these urls instead of just the ones below? If so, could you add a comment explaining it?

Reply:
so from the stuff i looked at, if you din't include this its throws errors. basically it broke the testing part. cause basically you are importing these and overriding with our own templates